scrapers/ep_votes/scrapers.py

```diff
@@ -351,12 +356,16 @@ def _url(self) -> str:
 
     def _extract_data(self) -> Optional[str]:
         rows = self._resource.select("#key_events-data .ep-table-row")
-        row = self._summary_row(rows)
+        summary_rows = [row for row in rows if self._is_summary_row(row) is True]
```

Comment:
Can we name this summary_rows_in_week or something like that?
I found it to be a bit confusing at first, since the filtering for the (presumably) correct summary already takes place in `is_summary_row()`